Client/core/CNickGen.h

```diff
@@ -9,8 +9,8 @@
 
 #pragma once
 
-#define NICKGEN_NUM_ADJECTIVES      1048
-#define NICKGEN_NUM_NOUNS           934
+#define NICKGEN_NUM_ADJECTIVES      1040
+#define NICKGEN_NUM_NOUNS           921
```

Comment:
you need replace `const char* const CNickGen::m_szAdjectives` with `std::array<const char*, SIZE_HERE>` to avoid all possible problems

Reply:
so it will be a compile time error in the future

Reply:
Thanks!